funding/manager.go

```diff
@@ -453,6 +457,9 @@ func (m *Manager) BatchChannelSetup(batch *order.Batch,
 				NodePubkey:         matchedOrder.NodeKey[:],
 				LocalFundingAmount: chanAmt,
 				FundingShim:        fundingShim,
+				PushSat: int64(
```

Comment:
Hmm, seems we should also change the way we communicate the channel size, no? As an example:
 * I want to buy a channel of 1 BTC inbound. 
 * I also want to commit 0.5 BTC of my own coins so I'm able to send and forward immediately using the channel. 
 * Therefore, I want the remote party to open a 1.5 BTC channel, as I'll be pushing over 0.5 BTC during execution into their account. 

AFAICT rn with this PR, we'll open a 1 BTC channel with 0.5 being pushed over, so same total amount, but I should only really pay the other party for that 0.5 BTC they commit.

Reply:
Ah, I hadn't considered this. But it totally makes sense that the self channel balance should be additive. I'm going to fix up the PR with that logic in mind.